### PR TITLE
Fix minor formatting nits in getting started page

### PR DESCRIPTION
**Pull request by mborgerson**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ you'll need to acquire them on your own.
 
 !!! attention
 
-    The XQEMU project does not endorse or promote piracy. We don't link to the
+    The XQEMU project does not endorse or promote piracy. We don't link to
     copyrighted files, or discuss how to acquire them. The best way to acquire
```
Comment (JayFoxRox) [inline docs/getting-started.md]: While we are at it: "best way" &rarr; "only legal way"?

File: docs/getting-started.md [hunk 2]
```diff
@@ -30,16 +30,16 @@ you'll need to acquire them on your own.
...
 If your MCPX dump has an MD5 of `196a5f59a13382c185636e691d6c323d`, you dumped
-it badly and it's a couple of bytes off. It should start with 0x33 0xC0 and end
-with 0x02 0xEE.
+it badly and it's a couple of bytes off. It should start with `0x33 0xC0` and end
+with `0x02 0xEE`.
 
 #### Flash ROM Image (aka BIOS)
 
-Xbox 1.0 compatible bios (cromwell, 3944, 4034, 4036, ...). You can use a retail
+Xbox 1.0 compatible bios (cromwell, 4034, 4036, ...). You can use a retail
```
Comment (JayFoxRox) [inline docs/getting-started.md]: "bios" &rarr; "BIOS" (make it consistent)

File: docs/getting-started.md [hunk 3]
```diff
@@ -30,16 +30,16 @@ you'll need to acquire them on your own.
     MD5 (mcpx_1.0.bin) = d49c52a4102f6df7bcf8d0617ac475ed
 
 If your MCPX dump has an MD5 of `196a5f59a13382c185636e691d6c323d`, you dumped
-it badly and it's a couple of bytes off. It should start with 0x33 0xC0 and end
-with 0x02 0xEE.
+it badly and it's a couple of bytes off. It should start with `0x33 0xC0` and end
+with `0x02 0xEE`.
 
 #### Flash ROM Image (aka BIOS)
```
Comment (JayFoxRox) [inline docs/getting-started.md]: Shall we also add " / Kernel" here?

File: docs/getting-started.md (hunk 1 above)
Comment (mborgerson) [reply docs/getting-started.md]: Agree

File: docs/getting-started.md (hunk 2 above)
Comment (mborgerson) [reply docs/getting-started.md]: agree

File: docs/getting-started.md (hunk 3 above)
Comment (mborgerson) [reply docs/getting-started.md]: Hmm.. well they aren't the same thing, but the layperson may refer to them this way so I'll add it

File: docs/getting-started.md [hunk 4]
```diff
@@ -20,26 +20,26 @@ you'll need to acquire them on your own.
...
-with 0x02 0xEE.
+it badly and it's a couple of bytes off. It should start with `0x33 0xC0` and end
+with `0x02 0xEE`.
 
-#### Flash ROM Image (aka BIOS)
+#### Flash ROM Image (aka BIOS/Kernel)
 
-Xbox 1.0 compatible bios (cromwell, 3944, 4034, 4036, ...). You can use a retail
+Xbox 1.0 compatible BIOS (cromwell, 4034, 4036, ...). You can use a retail
 or debug image. Just like a real Xbox, running an unmodified retail bios will
```
Comment (mborgerson) [inline docs/getting-started.md]: ah shit

File: docs/getting-started.md [hunk 5]
```diff
@@ -20,26 +20,26 @@ you'll need to acquire them on your own.
...
+it badly and it's a couple of bytes off. It should start with `0x33 0xC0` and end
+with `0x02 0xEE`.
 
-#### Flash ROM Image (aka BIOS)
+#### Flash ROM Image (aka BIOS/Kernel)
 
-Xbox 1.0 compatible bios (cromwell, 3944, 4034, 4036, ...). You can use a retail
-or debug image. Just like a real Xbox, running an unmodified retail bios will
+Xbox 1.0 compatible BIOS (cromwell, 4034, 4036, ...). You can use a retail
+or debug image. Just like a real Xbox, running an unmodified retail BIOS Will
```
Comment (JayFoxRox) [inline docs/getting-started.md]: "Will" &rarr; "will"

Comment (mborgerson) [reply docs/getting-started.md]: done

**Pull request merged**